pkg/parser/parser.go

```diff
@@ -567,6 +570,17 @@ func (p *Parser) parseBlockWithEndTokens(scope *scope, endTokens map[lexer.Token
 }
 
 func (p *Parser) advance() {
+	p.advanceOne()
+	for p.cur.Type == lexer.WS {
+		p.advanceOne()
+	}
+	inc := 2
+	for p.peek.Type == lexer.WS {
```

Comment:
I hope there is a better way for doing this than these two loops :(

Reply:
if `advanceOne` is already peeking, why do you need this here. :thinking:

Reply:
oh, sorry, github confused me with the diff. it is advanceTo() that has the same loop. ignore me.